### ISSUE TITLE
Doesn't give good error messages when not passing in a callback

### ISSUE THREAD

**Issue by elie222** [meteor/validated-method#15]
When failing validation when not passing a callback function, I get the following error message:
`Uncaught TypeError: callback is not a function(…)`

Once passing in a callback, I can print out a nice error message to the console.

**Comment by elie222** [comment]
This is the relevant package code: (validated-method.js:58)

```
    try {
      return this.connection.apply(this.name, [args], options, callback);
    } catch (err) {
      // Get errors from the stub in the same way as from the server-side method
      callback(err);
    }
```

**Comment by stubailo** [comment]
@elie222 what do you think would be the desired behavior - should we throw the error, log it to the console, or just silently ignore it?

I'm thinking throwing is the safest?

**Comment by elie222** [comment]
I think it should work in the same way that regular synchronous meteor
methods work. Return a result or throw an error. I don't think this package
returns results either, does it?

**Comment by stubailo** [comment]
It returns exactly the same thing that the regular Meteor.call does. I think you're right that it should do the same thing for errors as well, this was an oversight in the implementation.

The only intentional modification here is that if the client-side simulation throws an error, it now goes to the error callback rather than being thrown.

**Comment by stubailo** [comment]
Fixed: https://github.com/meteor/validated-method/commit/81072d93874cf20d123cc35f8e256967488a4503

Published in 0.2.1.

**Comment by elie222** [comment]
Cool. Thanks!
